app/src/main/java/com/f0x1d/logfox/extensions/notifications/NotificationExtensions.kt

```diff
@@ -0,0 +1,10 @@
+package com.f0x1d.logfox.extensions.notifications
+
+import android.content.Context
+import androidx.core.app.NotificationCompat
+import com.f0x1d.logfox.R
+
+fun NotificationCompat.Builder.applyPrimaryColor(context: Context): NotificationCompat.Builder {
+    color = context.getColor(R.color.color_primary)
```

Comment:
I think it will be better to get `colorPrimary` from attributes

Reply:
I don't think we can do this because the service's `context` doesn't have a theme applied and will fail to get the color

Reply:
Then i'd better apply it only on Android 11 and below